docs/1-getting-started/index.md

```diff
@@ -19,19 +19,23 @@ brew tap defenseunicorns/tap
 brew install zarf
 ```
 
-The above command detects your OS and system architecture and installs the correct Zarf CLI binary for your machine. Once the above command is entered, the CLI should be installed on your $PATH and ready for immediate use.
+The above command detects your OS and system architecture and installs the correct Zarf CLI binary for your machine. Once the above command is entered, the CLI should be installed on your `$PATH` and is ready for immediate use.
 
 ### Downloading a Prebuilt Binary from our GitHub Releases
 
-All [Zarf releases](https://github.com/defenseunicorns/zarf/releases) on GitHub include prebuilt binaries that you can download and use. We offer a small range of combinations of OS and architecture for you to choose from. Once downloaded, you can install the binary onto your $PATH by moving the binary to the `/usr/local/bin` directory:
+All [Zarf releases](https://github.com/defenseunicorns/zarf/releases) on GitHub include prebuilt binaries that you can download and use. We offer a small range of combinations of OS and architecture for you to choose from.
+
+On most Linux distributions, you can install the binary onto your `$PATH` by simply moving the downloaded binary to the `/usr/local/bin` directory:
 
 ```bash
 mv ./path/to/downloaded/{ZARF_FILE} /usr/local/bin/zarf
 ```
 
+On Windows or macOS, you can install the binary onto your `$PATH` by moving the downloaded binary to the desired directory and modifying the `$PATH` environment variable to include that directory.
+
 ### Building the CLI from Scratch
 
-If you want to build the CLI from scratch, you can do that too. Our local builds depend on [Go 1.19.x](https://golang.org/doc/install) and are built using [make](https://www.gnu.org/software/make/).
+If you want to build the CLI from scratch, you can do that too. Our local builds depend on [Go 1.19.x](https://golang.org/doc/install) and [Node 18.x](https://nodejs.org/en) and are built using [make](https://www.gnu.org/software/make/).
```

Comment:
If you build the CLI yourself, do you also need to build the `init-package` as well? If so `docker` is a build dep.

Reply:
Added a note down below - basically yes, and no - you can either checkout a tag to make changes from or rename the init package.